java.lang.NullPointerException: Cannot invoke "java.util.List.size()" because the return value of "io.vertx.sqlclient.RowSet.columnsNames()" is null

```
import jakarta.persistence.EntityManagerFactory;
import org.hibernate.reactive.mutiny.Mutiny;

@Inject
EntityManagerFactory emf;

public Uni<Integer> myFunc() {
  Mutiny.SessionFactory factory = emf.unwrap(Mutiny.SessionFactory.class);
  return factory.withSession(session -> { session.createNativeQuery(storedProcedureCall).getSingleResultOrNull()
}
```

The call errors out when the stored procedure does not return any result with 

> java.lang.NullPointerException: Cannot invoke "java.util.List.size()" because the return value of "io.vertx.sqlclient.RowSet.columnsNames()" is null

I've tested quickly with ORM and the following exception is thrown:
```
rg.hibernate.exception.GenericJDBCException: JDBC exception executing SQL [CREATE OR REPLACE FUNCTION PROC_INSERT_RECORD (    TEXT_PARAM IN varchar(255),    ID_PARAM IN bigint   )  RETURNS void AS $BODY$     BEGIN         insert into simplerecord (text, updated, id) values (TEXT_PARAM, localtimestamp, ID_PARAM);     END; $BODY$ LANGUAGE plpgsql;] [No results were returned by the query.] [n/a]
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:63)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:258)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:164)
	at org.hibernate.sql.results.jdbc.internal.AbstractResultSetAccess.getMetaData(AbstractResultSetAccess.java:36)
	at org.hibernate.sql.results.jdbc.internal.AbstractResultSetAccess.getColumnCount(AbstractResultSetAccess.java:52)
	at org.hibernate.query.results.ResultSetMappingImpl.resolve(ResultSetMappingImpl.java:193)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.resolveJdbcValuesSource(JdbcSelectExecutorStandardImpl.java:477)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:267)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sql.internal.NativeSelectQueryPlanImpl.performList(NativeSelectQueryPlanImpl.java:97)
	at org.hibernate.query.sql.internal.NativeQueryImpl.doList(NativeQueryImpl.java:623)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:427)
	at org.hibernate.query.spi.AbstractSelectionQuery.getSingleResultOrNull(AbstractSelectionQuery.java:595)
	at org.hibernate.reactive.CustomStoredProcedureSqlTest.prepareOrmFactory(CustomStoredProcedureSqlTest.java:116)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at io.vertx.junit5.RunTestOnContext.lambda$null$2(RunTestOnContext.java:154)
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:277)
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:259)
	at io.vertx.core.impl.EventLoopContext.lambda$runOnContext$0(EventLoopContext.java:43)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:566)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.postgresql.util.PSQLException: No results were returned by the query.
	at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:135)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:240)
	... 33 more
```

@gavinking This error is thrown by the Postgres JDBC driver, but we could just return null if the query doesn't return any value. Should we be consistent with Hibernate ORM or should we just make it work? I don't see any drawback in allowing it.

--- a/README.md
+++ b/README.md
@@ -38,11 +38,11 @@ Hibernate Reactive has been tested with:
 - MS SQL Server 2019
 - Oracle 21.3
 - [Hibernate ORM][] 6.3.1.Final
-- [Vert.x Reactive PostgreSQL Client](https://vertx.io/docs/vertx-pg-client/java/) 4.4.6
-- [Vert.x Reactive MySQL Client](https://vertx.io/docs/vertx-mysql-client/java/) 4.4.6
-- [Vert.x Reactive Db2 Client](https://vertx.io/docs/vertx-db2-client/java/) 4.4.6
-- [Vert.x Reactive MS SQL Server Client](https://vertx.io/docs/vertx-mssql-client/java/) 4.4.6
-- [Vert.x Reactive Oracle Client](https://vertx.io/docs/vertx-oracle-client/java/) 4.4.6
+- [Vert.x Reactive PostgreSQL Client](https://vertx.io/docs/vertx-pg-client/java/) 4.4.5
+- [Vert.x Reactive MySQL Client](https://vertx.io/docs/vertx-mysql-client/java/) 4.4.5
+- [Vert.x Reactive Db2 Client](https://vertx.io/docs/vertx-db2-client/java/) 4.4.5
+- [Vert.x Reactive MS SQL Server Client](https://vertx.io/docs/vertx-mssql-client/java/) 4.4.5
+- [Vert.x Reactive Oracle Client](https://vertx.io/docs/vertx-oracle-client/java/) 4.4.5
 - [Quarkus][Quarkus] via the Hibernate Reactive extension
 
 [PostgreSQL]: https://www.postgresql.org

--- a/build.gradle
+++ b/build.gradle
@@ -83,7 +83,7 @@ ext {
     // Example:
     // ./gradlew build -PvertxSqlClientVersion=4.0.0-SNAPSHOT
     if ( !project.hasProperty( 'vertxSqlClientVersion' ) ) {
-        vertxSqlClientVersion = '4.4.6'
+        vertxSqlClientVersion = '4.4.5'
     }
 
     testcontainersVersion = '1.18.3'

--- a/gradle.properties
+++ b/gradle.properties
@@ -47,9 +47,9 @@ org.gradle.java.installations.auto-download=false
 #skipOrmVersionParsing = true
 
 # Override default Vert.x Sql client version
-#vertxSqlClientVersion = 4.4.6-SNAPSHOT
+#vertxSqlClientVersion = 4.4.5-SNAPSHOT
 
 # Override default Vert.x Web client and server versions. For integration tests, both default to vertxSqlClientVersion
-#vertxWebVersion = 4.4.6
-#vertxWebtClientVersion = 4.4.6
+#vertxWebVersion = 4.4.5
+#vertxWebtClientVersion = 4.4.5
 

--- a/hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java
+++ b/hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java
@@ -28,9 +28,9 @@ import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.util.Calendar;
+import java.util.List;
 import java.util.Map;
 import java.util.NoSuchElementException;
-import java.util.Objects;
 import java.util.function.Function;
 
 import org.hibernate.engine.jdbc.BlobProxy;
@@ -41,6 +41,9 @@ import io.vertx.sqlclient.Row;
 import io.vertx.sqlclient.RowIterator;
 import io.vertx.sqlclient.RowSet;
 import io.vertx.sqlclient.desc.ColumnDescriptor;
+
+import static java.util.Collections.emptyList;
+import static java.util.Objects.requireNonNull;
 
 /**
  * An adaptor that allows Hibernate core code which expects a JDBC
@@ -54,6 +57,7 @@ public class ResultSetAdaptor implements ResultSet {
 	private boolean wasNull;
 
 	public ResultSetAdaptor(RowSet<Row> rows) {
+		requireNonNull( rows );
 		this.iterator = rows.iterator();
 		this.rows = rows;
 	}
@@ -199,13 +203,24 @@ public class ResultSetAdaptor implements ResultSet {
 	}
 
 	private <T> T caseInsensitiveGet(String columnLabel, Function<String, T> produce) {
-		for ( String columnName : rows.columnsNames() ) {
+		for ( String columnName : getColumnsNames() ) {
 			if ( columnName.equalsIgnoreCase( columnLabel ) ) {
 				return produce.apply( columnName );
 			}
 		}
+
 		// Same error thrown by io.vertx.sqlclient.Row when it doesn't find the label
 		throw new NoSuchElementException( "Column " + columnLabel + " does not exist" );
+	}
+
+	/**
+	 * rows.columnsNames() might return null for some databases.
+	 * For example, when creating a stored procedure in PostgreSQL using a native query.
+	 *
+	 * @return A list of column names or an empty list.
+	 */
+	private List<String> getColumnsNames() {
+		return rows.columnsNames() == null ? emptyList() : rows.columnsNames();
 	}
 
 	@Override
@@ -368,7 +383,7 @@ public class ResultSetAdaptor implements ResultSet {
 		return new ResultSetMetaData() {
 			@Override
 			public int getColumnCount() {
-				return rows.columnsNames().size();
+				return getColumnsNames() == null ? 0 : getColumnsNames().size();
 			}
 
 			@Override
@@ -379,12 +394,12 @@ public class ResultSetAdaptor implements ResultSet {
 
 			@Override
 			public String getColumnLabel(int column) {
-				return rows.columnsNames().get( column - 1 );
+				return getColumnsNames().get( column - 1 );
 			}
 
 			@Override
 			public String getColumnName(int column) {
-				return rows.columnsNames().get( column - 1 );
+				return getColumnsNames().get( column - 1 );
 			}
 
 			@Override
@@ -502,7 +517,7 @@ public class ResultSetAdaptor implements ResultSet {
 	public int findColumn(String columnLabel) {
 		// JDBC parameters index start from 1
 		int index = 1;
-		for ( String column : rows.columnsNames() ) {
+		for ( String column : getColumnsNames() ) {
 			// Some dbs, like Oracle and Db2, return the column names always in uppercase
 			if ( column.equalsIgnoreCase( columnLabel ) ) {
 				return index;
@@ -865,7 +880,7 @@ public class ResultSetAdaptor implements ResultSet {
 		private final Buffer buffer;
 
 		private RowIdAdaptor(Buffer buffer) {
-			Objects.requireNonNull( buffer );
+			requireNonNull( buffer );
 			this.buffer = buffer;
 		}
 

--- a/hibernate-reactive-core/src/main/java/org/hibernate/reactive/logging/impl/Log.java
+++ b/hibernate-reactive-core/src/main/java/org/hibernate/reactive/logging/impl/Log.java
@@ -255,6 +255,9 @@ public interface Log extends BasicLogger {
 	@Message(id = 79, value = "The configuration property '%1$s' was not provided, or is in invalid format. This is required when using the default DefaultSqlClientPool: either provide the configuration setting or integrate with a different SqlClientPool implementation")
 	HibernateException blankConnectionString(String property);
 
+	@Message(id = 80, value = "No results were returned by the query (you can try running it with '.executeUpdate()'): %1$s")
+	HibernateException noResultException(String sql);
+
 	// Same method that exists in CoreMessageLogger
 	@LogMessage(level = WARN)
 	@Message(id = 104, value = "firstResult/maxResults specified with collection fetch; applying in memory!" )

--- a/hibernate-reactive-core/src/main/java/org/hibernate/reactive/sql/results/internal/ReactiveDeferredResultSetAccess.java
+++ b/hibernate-reactive-core/src/main/java/org/hibernate/reactive/sql/results/internal/ReactiveDeferredResultSetAccess.java
@@ -10,6 +10,7 @@ import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
+import java.util.concurrent.CompletionException;
 import java.util.concurrent.CompletionStage;
 import java.util.function.Function;
 
@@ -24,6 +25,7 @@ import org.hibernate.reactive.logging.impl.LoggerFactory;
 import org.hibernate.reactive.pool.ReactiveConnection;
 import org.hibernate.reactive.pool.impl.Parameters;
 import org.hibernate.reactive.session.ReactiveConnectionSupplier;
+import org.hibernate.reactive.util.impl.CompletionStages;
 import org.hibernate.resource.jdbc.spi.JdbcSessionContext;
 import org.hibernate.resource.jdbc.spi.LogicalConnectionImplementor;
 import org.hibernate.sql.exec.spi.ExecutionContext;
@@ -42,7 +44,6 @@ public class ReactiveDeferredResultSetAccess extends DeferredResultSetAccess imp
 
 	private static final Log LOG = LoggerFactory.make( Log.class, MethodHandles.lookup() );
 	private final SqlStatementLogger sqlStatementLogger;
-	private final Function<String, PreparedStatement> statementCreator;
 
 	private final ExecutionContext executionContext;
 
@@ -60,7 +61,6 @@ public class ReactiveDeferredResultSetAccess extends DeferredResultSetAccess imp
 		super( jdbcSelect, jdbcParameterBindings, executionContext, statementCreator );
 		this.executionContext = executionContext;
 		this.sqlStatementLogger = executionContext.getSession().getJdbcServices().getSqlStatementLogger();
-		this.statementCreator = statementCreator;
 	}
 
 	@Override
@@ -163,15 +163,31 @@ public class ReactiveDeferredResultSetAccess extends DeferredResultSetAccess imp
 					eventListenerManager.jdbcExecuteStatementStart();
 					return connection()
 							.selectJdbc( sql, parameters )
+							.thenCompose( this::validateResultSet )
 							.whenComplete( (resultSet, throwable) -> {
 								// FIXME: I don't know if this event makes sense for Vert.x
 								eventListenerManager.jdbcExecuteStatementEnd();
 								sqlStatementLogger.logSlowQuery( getFinalSql(), executeStartNanos, context() );
 							} )
 							.thenCompose( this::reactiveSkipRows )
-							.handle( this::convertException );
+							.handle( CompletionStages::handle )
+							.thenCompose( handler -> handler.hasFailed()
+										? convertException( resultSet, handler.getThrowable() )
+										: handler.getResultAsCompletionStage()
+							);
 				} )
 				.whenComplete( (o, throwable) -> logicalConnection.afterStatement() );
+	}
+
+	private CompletionStage<ResultSet> validateResultSet(ResultSet resultSet) {
+		try {
+			return resultSet.getMetaData().getColumnCount() == 0
+					? failedFuture( LOG.noResultException( getFinalSql() ) )
+					: completedFuture( resultSet );
+		}
+		catch (SQLException e) {
+			throw new RuntimeException( e );
+		}
 	}
 
 	private ResultSet saveResultSet(ResultSet resultSet) {
@@ -189,18 +205,29 @@ public class ReactiveDeferredResultSetAccess extends DeferredResultSetAccess imp
 		return ( (ReactiveConnectionSupplier) executionContext.getSession() ).getReactiveConnection();
 	}
 
-	private ResultSet convertException(ResultSet resultSet, Throwable throwable) {
-		// FIXME: Vert.x will probably throw another exception. Check this.
-		if ( throwable instanceof SQLException) {
-			throw executionContext.getSession().getJdbcServices()
-					.getSqlExceptionHelper()
-					// FIXME: Add this to the logger?
-					.convert( (SQLException) throwable, "Exception executing SQL [" + getFinalSql() + "]" );
-		}
+	private <T> CompletionStage<T> convertException(T object, Throwable throwable) {
 		if ( throwable != null ) {
-			throw new HibernateException( throwable );
+			Throwable cause = throwable;
+			if ( throwable instanceof CompletionException ) {
+				cause = throwable.getCause();
+			}
+			// I doubt this is ever going to happen because Vert.x is not going to throw an SQLException
+			if ( cause instanceof SQLException ) {
+				return failedFuture( executionContext
+											 .getSession().getJdbcServices()
+											 .getSqlExceptionHelper()
+											 .convert(
+													 (SQLException) cause,
+													 "Exception executing SQL [" + getFinalSql() + "]"
+											 )
+				);
+			}
+			if ( cause instanceof HibernateException ) {
+				return failedFuture( cause );
+			}
+			return failedFuture( new HibernateException( cause ) );
 		}
-		return resultSet;
+		return completedFuture( object );
 	}
 
 	private CompletionStage<ResultSet> reactiveSkipRows(ResultSet resultSet) {

--- a/hibernate-reactive-core/src/test/java/org/hibernate/reactive/CustomOneToOneStoredProcedureSqlTest.java
+++ b/hibernate-reactive-core/src/test/java/org/hibernate/reactive/CustomOneToOneStoredProcedureSqlTest.java
@@ -9,9 +9,11 @@ import io.vertx.junit5.Timeout;
 import io.vertx.junit5.VertxTestContext;
 import jakarta.persistence.*;
 
+import org.hibernate.HibernateException;
 import org.hibernate.annotations.SQLDelete;
 import org.hibernate.annotations.SQLInsert;
 import org.hibernate.reactive.testing.DBSelectionExtension;
+import org.hibernate.reactive.testing.ReactiveAssertions;
 
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
@@ -23,6 +25,7 @@ import java.util.Collection;
 import java.util.List;
 
 import static java.util.concurrent.TimeUnit.*;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.*;
 import static org.hibernate.reactive.testing.DBSelectionExtension.*;
 import static org.junit.jupiter.api.Assertions.*;
@@ -81,6 +84,22 @@ public class CustomOneToOneStoredProcedureSqlTest extends BaseReactiveTest {
 						.thenCompose( v -> s.persist( individualPerson, driverLicence ) )
 						.thenCompose( v -> s.flush() )
 				)
+		);
+	}
+
+	@Test
+	public void testFailureWithGetSingleResultOrNull(VertxTestContext context) {
+		test( context, ReactiveAssertions.assertThrown( HibernateException.class, getMutinySessionFactory()
+						.withTransaction( s ->  s.createNativeQuery( INSERT_DRIVER_LICENCE_SQL ).getSingleResultOrNull() ) )
+				.invoke( e -> assertThat( e ).hasMessageContainingAll( "HR000080:", INSERT_DRIVER_LICENCE_SQL ) )
+		);
+	}
+
+	@Test
+	public void testFailureWithGetSingleResult(VertxTestContext context) {
+		test( context, ReactiveAssertions.assertThrown( HibernateException.class, getSessionFactory()
+						.withTransaction( s ->  s.createNativeQuery( INSERT_DRIVER_LICENCE_SQL ).getSingleResult() ) )
+				.thenAccept( e -> assertThat( e ).hasMessageContainingAll( "HR000080:", INSERT_DRIVER_LICENCE_SQL ) )
 		);
 	}
 

--- a/hibernate-reactive-core/src/test/java/org/hibernate/reactive/CustomStoredProcedureSqlTest.java
+++ b/hibernate-reactive-core/src/test/java/org/hibernate/reactive/CustomStoredProcedureSqlTest.java
@@ -9,10 +9,12 @@ import java.time.LocalDateTime;
 import java.util.Collection;
 import java.util.List;
 
+import org.hibernate.HibernateException;
 import org.hibernate.annotations.SQLDelete;
 import org.hibernate.annotations.SQLInsert;
 import org.hibernate.annotations.SQLUpdate;
 import org.hibernate.reactive.testing.DBSelectionExtension;
+import org.hibernate.reactive.testing.ReactiveAssertions;
 
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
@@ -29,6 +31,7 @@ import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 
 import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
 import static org.hibernate.reactive.testing.DBSelectionExtension.runOnlyFor;
 import static org.junit.jupiter.api.Assertions.assertEquals;
@@ -99,6 +102,22 @@ public class CustomStoredProcedureSqlTest extends BaseReactiveTest {
 						.thenCompose( v -> s.persist( theRecord ) )
 						.thenCompose( v -> s.flush() )
 				)
+		);
+	}
+
+	@Test
+	public void testFailureWithGetSingleResultOrNull(VertxTestContext context) {
+		test( context, ReactiveAssertions.assertThrown( HibernateException.class, getMutinySessionFactory()
+				.withTransaction( s ->  s.createNativeQuery( INSERT_SP_SQL ).getSingleResultOrNull() ) )
+				.invoke( e -> assertThat( e ).hasMessageContainingAll( "HR000080:", INSERT_SP_SQL ) )
+		);
+	}
+
+	@Test
+	public void testFailureWithGetSingleResult(VertxTestContext context) {
+		test( context, ReactiveAssertions.assertThrown( HibernateException.class, getSessionFactory()
+						.withTransaction( s ->  s.createNativeQuery( INSERT_SP_SQL ).getSingleResult() ) )
+				.thenAccept( e -> assertThat( e ).hasMessageContainingAll( "HR000080:", INSERT_SP_SQL ) )
 		);
 	}
 

--- a/tooling/jbang/CockroachDBReactiveTest.java.qute
+++ b/tooling/jbang/CockroachDBReactiveTest.java.qute
@@ -5,8 +5,8 @@
  * Copyright: Red Hat Inc. and Hibernate Authors
  */
 
-//DEPS io.vertx:vertx-pg-client:$\{vertx.version:4.4.6}
-//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.6}
+//DEPS io.vertx:vertx-pg-client:$\{vertx.version:4.4.5}
+//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:$\{hibernate-reactive.version:2.0.5.Final}
 //DEPS org.assertj:assertj-core:3.24.2
 //DEPS junit:junit:4.13.2

--- a/tooling/jbang/Db2ReactiveTest.java.qute
+++ b/tooling/jbang/Db2ReactiveTest.java.qute
@@ -5,8 +5,8 @@
  * Copyright: Red Hat Inc. and Hibernate Authors
  */
 
-//DEPS io.vertx:vertx-db2-client:$\{vertx.version:4.4.6}
-//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.6}
+//DEPS io.vertx:vertx-db2-client:$\{vertx.version:4.4.5}
+//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:$\{hibernate-reactive.version:2.0.5.Final}
 //DEPS org.assertj:assertj-core:3.24.2
 //DEPS junit:junit:4.13.2

--- a/tooling/jbang/Example.java
+++ b/tooling/jbang/Example.java
@@ -6,9 +6,9 @@
  */
 
 //DEPS com.ongres.scram:client:2.1
-//DEPS io.vertx:vertx-pg-client:${vertx.version:4.4.6}
-//DEPS io.vertx:vertx-mysql-client:${vertx.version:4.4.6}
-//DEPS io.vertx:vertx-db2-client:${vertx.version:4.4.6}
+//DEPS io.vertx:vertx-pg-client:${vertx.version:4.4.5}
+//DEPS io.vertx:vertx-mysql-client:${vertx.version:4.4.5}
+//DEPS io.vertx:vertx-db2-client:${vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:${hibernate-reactive.version:2.0.5.Final}
 //DEPS org.slf4j:slf4j-simple:2.0.7
 //DESCRIPTION Allow authentication to PostgreSQL using SCRAM:

--- a/tooling/jbang/MariaDBReactiveTest.java.qute
+++ b/tooling/jbang/MariaDBReactiveTest.java.qute
@@ -5,8 +5,8 @@
  * Copyright: Red Hat Inc. and Hibernate Authors
  */
 
-//DEPS io.vertx:vertx-mysql-client:$\{vertx.version:4.4.6}
-//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.6}
+//DEPS io.vertx:vertx-mysql-client:$\{vertx.version:4.4.5}
+//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:$\{hibernate-reactive.version:2.0.5.Final}
 //DEPS org.assertj:assertj-core:3.24.2
 //DEPS junit:junit:4.13.2

--- a/tooling/jbang/MySQLReactiveTest.java.qute
+++ b/tooling/jbang/MySQLReactiveTest.java.qute
@@ -5,8 +5,8 @@
  * Copyright: Red Hat Inc. and Hibernate Authors
  */
 
-//DEPS io.vertx:vertx-mysql-client:$\{vertx.version:4.4.6}
-//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.6}
+//DEPS io.vertx:vertx-mysql-client:$\{vertx.version:4.4.5}
+//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:$\{hibernate-reactive.version:2.0.5.Final}
 //DEPS org.assertj:assertj-core:3.24.2
 //DEPS junit:junit:4.13.2

--- a/tooling/jbang/PostgreSQLReactiveTest.java.qute
+++ b/tooling/jbang/PostgreSQLReactiveTest.java.qute
@@ -5,8 +5,8 @@
  * Copyright: Red Hat Inc. and Hibernate Authors
  */
 
-//DEPS io.vertx:vertx-pg-client:$\{vertx.version:4.4.6}
-//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.6}
+//DEPS io.vertx:vertx-pg-client:$\{vertx.version:4.4.5}
+//DEPS io.vertx:vertx-unit:$\{vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:$\{hibernate-reactive.version:2.0.5.Final}
 //DEPS org.assertj:assertj-core:3.24.2
 //DEPS junit:junit:4.13.2

--- a/tooling/jbang/ReactiveTest.java
+++ b/tooling/jbang/ReactiveTest.java
@@ -5,11 +5,11 @@
  */
 
 ///usr/bin/env jbang "$0" "$@" ; exit $?
-//DEPS io.vertx:vertx-pg-client:${vertx.version:4.4.6}
+//DEPS io.vertx:vertx-pg-client:${vertx.version:4.4.5}
 //DEPS com.ongres.scram:client:2.1
-//DEPS io.vertx:vertx-db2-client:${vertx.version:4.4.6}
-//DEPS io.vertx:vertx-mysql-client:${vertx.version:4.4.6}
-//DEPS io.vertx:vertx-unit:${vertx.version:4.4.6}
+//DEPS io.vertx:vertx-db2-client:${vertx.version:4.4.5}
+//DEPS io.vertx:vertx-mysql-client:${vertx.version:4.4.5}
+//DEPS io.vertx:vertx-unit:${vertx.version:4.4.5}
 //DEPS org.hibernate.reactive:hibernate-reactive-core:${hibernate-reactive.version:2.0.5.Final}
 //DEPS org.assertj:assertj-core:3.24.2
 //DEPS junit:junit:4.13.2

